vars/withDefinitionImportSecretsAndEnvVars.groovy

```diff
@@ -1,5 +1,5 @@
 
-def call(String product, String environment, Map<String, String> vaultOverrides, Closure body) {
+def call(String product, String environment, String highLevelDataSetupKeyVaultName, Map<String, String> vaultOverrides, Closure body) {
```

Comment:
please refactor this to just pass the vault name in as the product, then everything should work.